pkg/internal/codegen/parse/util.go

```diff
@@ -336,11 +336,13 @@ func parseByteValue(b []byte) string {
 
 // parseDescription parse comments above each field in the type definition.
 func parseDescription(res []string) string {
-	var temp, data string
+	var temp strings.Builder
+	var data string
```

Comment:
Consider renaming `data` to `desc` to reflect the usage ?

Reply:
Done!